### PR TITLE
Changing plugin version number

### PR DESCRIPTION
**Pull request by allenbross-wf**
### Changes:

- Changed plugin version from 3.0 to 3.2.

### Pull request process

- [ ] Full description of changes provided above.
- [ ] Reviewed by another person. (+1)
- [ ] If this is a code change, reviewed by a second person. (+1)
- [ ] QA'd to the specifications listed [here](https://github.com/DataQualityCommittee/dqc_us_rules#quality-assurance-qa-of-a-pull-request). Documentation only changes do not require the formal code quality assurance process. (+10)

### CODE REVIEW

File: dqc_us_rules/__init__.py [hunk 1]
```diff
@@ -59,7 +59,7 @@ def _plugins_to_run(mod, include_start=True):
 
 __pluginInfo__ = {
     'name': 'DQC.SEC.ALL',
-    'version': '3.0',
+    'version': '3.2',
```
Comment (mindeerickson-wf) [inline dqc_us_rules/__init__.py]: Did we skip 3.1?

Comment (allenbross-wf) [inline dqc_us_rules/__init__.py]: Yes.  I had not remembered to change the plugin version when I updated the rule version.  This change gets them in sync.

**Comment by joyreistad-wf** [comment]
+1 if version doesn't need to be changed to 3.1

File: dqc_us_rules/__init__.py (hunk 1 above)
Comment (mindeerickson-wf) [inline dqc_us_rules/__init__.py]: Perfect.  Thanks!

**Comment by mindeerickson-wf** [comment]
+1

**Comment by chrislococo-wf** [comment]
@allenbross-wf Plugin version != library version. Library is still 3.1.0

Additionally, can we start calling it 3.2.0 in the plugin or is there a convention against that I am not aware of?

**Comment by chrislococo-wf** [comment]
+1

**Comment by joyreistad-wf** [comment]
1

**Comment by chrislococo-wf** [comment]
+10

Tests pass cleanly, pinned versions in both locations match.

**Comment by andrewperkins-wf** [comment]
Reviewed, QA'ed, merging.

**Pull request merged**